IndicoV2.V1Adapters/Client/Submissions/IndicoSubmissionsAdapter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Indico;
+using Indico.Mutation;
+using IndicoV2.Client.Submissions;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using IndicoV2.Client.Submissions.Models;
+using IndicoV2.V1Adapters.Client.Submissions.Models;
+
+namespace IndicoV2.V1Adapters.Client.Submissions
+{
+    public class IndicoSubmissionsAdapter : ISubmissionsClient
+    {
+        private readonly IndicoClient indicoClient;
```

Comment:
These changes allows us to use existing solution. Adapter exposes its interface but existing implementation is still in use.
Moreover we can later insert `GraphQlHttpClient` instead of `IndicoClient`, so we will be ale to provide new implementation piece by piece.